pbsmetrics/config/metrics_test.go

```diff
@@ -86,6 +86,27 @@ func TestMultiMetricsEngine(t *testing.T) {
 		metricsEngine.RecordAdapterBidReceived(pubLabels, openrtb_ext.BidTypeBanner, true)
 		metricsEngine.RecordAdapterTime(pubLabels, time.Millisecond*20)
 	}
+	labels_blacklist := []pbsmetrics.Labels{
+		{
+			Source:        pbsmetrics.DemandWeb,
+			RType:         pbsmetrics.ReqTypeAMP,
+			PubID:         "test2",
+			Browser:       pbsmetrics.BrowserOther,
+			CookieFlag:    pbsmetrics.CookieFlagYes,
+			RequestStatus: pbsmetrics.RequestStatusBlacklisted,
+		},
+		{
+			Source:        pbsmetrics.DemandWeb,
+			RType:         pbsmetrics.ReqTypeVideo,
+			PubID:         "test2",
+			Browser:       pbsmetrics.BrowserOther,
+			CookieFlag:    pbsmetrics.CookieFlagYes,
+			RequestStatus: pbsmetrics.RequestStatusBlacklisted,
+		},
+	}
+	for i := 0; i < 2; i++ {
```

Comment:
Rather than hard coding `2` as the end of the for loop, we should rather just do:
```
for _, label := range blacklistLabels {
    ...
}
```

Reply:
Done